inference-engine/thirdparty/clDNN/src/graph_optimizer/add_onednn_optimization_attributes.cpp

```diff
@@ -6,13 +6,32 @@
 
 #include "pass_manager.h"
 #include "program_node.h"
+#include "fully_connected_inst.h"
+#include <impls/onednn/utils.hpp>
 
 using namespace cldnn;
 
 void add_onednn_optimization_attributes::run(program& p) {
 #ifdef ENABLE_ONEDNN_FOR_GPU
     for (auto& node : p.get_processing_order()) {
         if (node->get_preferred_impl_type() == impl_types::onednn) {
+            if (node->is_type<fully_connected>()) {
+                auto fc_prim = node->as<fully_connected>().get_primitive();
+
+                // Reshape fused ops tensors for OneDNN FC if needed
+                if (fc_prim->input_size == 3) {
+                    for (auto& fused_prim : node->get_fused_primitives()) {
+                        auto fused_node = fused_prim.node;
```

Comment:
[random spot] could you add unit test for such case?

Reply:
I've added several fusings test cases